tools/recipes/utils.sh

```diff
@@ -32,6 +33,14 @@ install_readme() {
 
     cat "$LISA_ASSET_RECIPE" >> "$src_info"
 
+    # If compiled with musl-libc, add the COPYRIGHT
+    if [ -n ${musl_libc:+x} ]
+    then
+        printf "\n\nThe sources were compiled with musl-libc (content of COPYRIGHT):\n\n" >> "$src_info"
+        wget https://git.musl-libc.org/cgit/musl/plain/COPYRIGHT
+        cat COPYRIGHT >> "$src_info"
```

Comment:
This would do the same without potentially overriding an existing COPYRIGHT file:
```
wget https://git.musl-libc.org/cgit/musl/plain/COPYRIGHT -O - >> "$src_info"
```